### ISSUE TITLE
Option to fetch and keep ALL articles, and keep articles older than a month

### ISSUE THREAD

**Issue by advait404** [yang991178/fluent-reader-lite#72]
I think that there should be an option to fetch all articles from the server, and not remove those that are more than 1 month old. The app deletes older articles and I find this to be inconvenient, especially since there is already a max article limit of 1500.

The user should be able to choose the article keep time, and this is especially important for new users, since they may import their feeds to fluent reader, see that there are less articles, and think that the app does not work properly. Making the article keep time to indefinite and having an max article limit might be better.

**Comment by mrg358** [comment]
Yes to this idea. I would like to read articles older then a month. Current setting is very limiting. I think we can setup the limits we want in our backend service anyway. There are many sources with old but gold content. Please make it available to read them with Fluent Reader too. Thank you for all your work on this app.

**Comment by Nyaacinth** [comment]
Retaining old articles will help Fluent Reader Lite and Fluent Reader to relay each other, so that the experience on both sides does not feel separated. It would be great to add this feature.

**Comment by Nyaacinth** [comment]
At least I think this is a feature that can be easily added. Sometimes when fetching articles, it will unexpectedly get all the articles (includes articles older than 28 days, like 1149d) and display it, and keep it until next refresh.
